article/app/views/liveblog/liveBlogBodyContent.scala.html

```diff
@@ -0,0 +1,44 @@
+@import _root_.liveblog.KeyEventData
+@import model.Badges.badgeFor
+
+@import _root_.liveblog.Canonical
```

Comment:
it probably wasn't yours originally, but the line spacing is a little wonky - and actually there's no need for any of the imports in the signature by the looks, so they could all be below (or all above if you prefer that way)